### DISCUSSION TITLE
Entity Linking without an alias match

### DISCUSSION THREAD

**Discussion by commahawk** [explosion/spaCy#13421] [Help: Other Questions]
From what I can tell in the code, the `get_candidates` call for Entity Linking to the KB requires that the entity match an alias. It seems possible, and maybe desirable to surface candidates with either an alias OR a vector with low distance (or maybe a waterfall from alias to vector matches). Then you have something not unlike a RAG approach. Is that a good idea or should I be doing more to produce a smaller set or entities at the NER layer so I can have aliases for every entity?

**Comment by svlandeg** [comment]
Hi!

You can definitely overwrite the custom behaviour of the `get_candidates` funcationality. See also [this thread](https://github.com/explosion/spaCy/discussions/7801) for more details!
